srcpkgs/foo2zjs/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'foo2zjs'
+pkgname=foo2zjs
+version=20171202
+revision=1
+wrksrc="foo2zjs"
+build_style=gnu-makefile
+hostmakedepends="pkg-config"
+makedepends="ghostscript foomatic-db-engine"
+depends="ghostscript foomatic-db-engine"
+short_desc="Collection of print drivers by rkkda.com"
+maintainer="Jasper Chan <jasperchan515@gmail.com>"
+license="GPL-2"
+homepage="foo2zjs.rkkda.com"
+distfiles="http://foo2zjs.rkkda.com/foo2zjs.tar.gz"
```

Comment:
eh ? shouldn't it need a version ?

Reply:
They don't seem to host older versions as far as I can tell.
From the install guide:

Click the link, or cut and paste the whole command line below to download the driver. 
``` bash
$ wget -O foo2zjs.tar.gz http://foo2zjs.rkkda.com/foo2zjs.tar.gz
```

Reply:
The version number is from the changelog at http://foo2zjs.rkkda.com/ChangeLog